src/components/Album.jsx

```diff
@@ -7,7 +7,7 @@ export const Album = ({ album }) => {
 
   return (
     <div className="album-container">
-            <CoverImage img={album.images[1].url} alt={album.name} />
+      <CoverImage img={album.images[1].url} alt={album.name} />
```

Comment:
actually can we change this so we are displaying the image size that is `640` in size? we can use `find` here

Reply:
If we choose to display the image that is in size 640, it will be too big compared to the example they provided as well as the design. Or do you have another idea how to work around that?

Reply:
this should handled by the `CSS` styling we have that sets the size so we can use any image size here.

Reply:
Ok, just seems uneccessary when the one in use now is perfect size, but I'll fix it!

Reply:
Yeah, but one of the other changes is to change the max-width of the album container from a fixed size of 300px to auto resize up to 600px, so we might want a better quality image instead of upsizing a 300px one.

Reply:
Either, we probably should be a bit more explicit here than just using the first element of the array. In this case it doesn't matter since we're using static data, but for example, we are pulling real data from an API, we are not sure that the first element of the array will always have the value that we want. Using the find to match the image size that we want would help add a bit more protection for those kind of cases.

Reply:
I commited the change, but the props need to be fixed.

Reply:
And thank you for the explanation! It made it all a bit clearer to me :)